src/migrations/migrations/migrateNotificationSettingsToV2.ts

```diff
@@ -0,0 +1,28 @@
+import { Migration, MigrationName } from '@/migrations/types';
+import {
+  getAllNotificationSettingsFromStorage,
+  getSettingsVersion,
+  setAllNotificationSettingsToStorage,
+  setSettingsVersion,
+} from '@/notifications/settings/storage';
+
+export default function migrateNotificationSettingsToV2(): Migration {
+  return {
+    name: MigrationName.migrateNotificationSettingsToVersion2,
+    migrate(): Promise<void> {
+      const walletSettings = getAllNotificationSettingsFromStorage();
+
+      // migrate to V2
+      if (walletSettings.length && getSettingsVersion() === 1) {
+        //
+        const newSettings = walletSettings.map(wallet => ({
+          ...wallet,
+          appliedDefaults: true,
+        }));
+        setAllNotificationSettingsToStorage(newSettings);
+      }
+      setSettingsVersion(2);
```

Comment:
I think you're right. I kept it because at first it wasn't in the new migrations and I somehow got fixated on having a version

Reply:
I think we may need it. SO when you install the app after we ship this, wouldn't the migration still run? If so we want to have an easy way to tell whether we actually need to do something in this migration. If the initialized settings are new already and we have all `appliedDefaults: false` that means we need to call firebase to subscribe to topics. If we run this migration without the versions we will always run it.
Am I wrong here?

Reply:
Yeah it would run on first open, but there would be no accounts to migrate, and the migration would only run once.

Reply:
I don't think we need this. If we DO need this mechanism, I'd like to brainstorm how we could better link what "v2" is with the settings objects themselves. Having it as a separate value could cause issues in the future e.g. we add a v3, something goes wrong, now settings object and settings versions are out of sync.

Reply:
I think you are right, I can get rid of this and instead of having a version state I can check whether there are any wallets and if there are any wallets I can check whether the first one has a `successfullyFinishedInitialSubscription` specified which will tell us it's v2